packages/widgetbook/lib/src/fields/num_slider_field.dart

```diff
@@ -28,23 +28,37 @@ class NumSliderField<T extends num> extends Field<T> {
   /// The number of discrete divisions in the slider.
   final int? divisions;
 
+  Size _getTextSize(String text, TextStyle style) {
+    final textPainter = TextPainter(
+      text: TextSpan(text: text, style: style),
+      maxLines: 1,
+      textDirection: TextDirection.ltr,
+    )..layout();
+    return textPainter.size;
+  }
+
   @override
   Widget toWidget(BuildContext context, String group, T? value) {
     final defaultValue = (T == int ? 0 : 0.0) as T;
+    final label = codec.toParam(value ?? initialValue ?? defaultValue);
+    final maxLabel = codec.toParam(max);
 
     return Row(
       mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      spacing:
+          SliderTheme.of(context).thumbShape == SliderComponentShape.noThumb
```

Comment:
This should be `overlayShape`, not `thumbShape`, to match the value use in the `ThemeData`.

```suggestion
          SliderTheme.of(context).overlayShape == SliderComponentShape.noThumb
```